test/epictest/three_legged_cat_test.go

```diff
@@ -71,7 +71,7 @@ func RunThreeLeggedCat(data []byte, conf testutil.LatencyConfig) error {
 		return err
 	}
 
-	readerCatted, err := coreunix.Cat(catter, keyAdded)
+	readerCatted, err := coreunix.Cat(catter, keyAdded.String())
```

Comment:
yeah, the first two are right, the third isnt... I see your point and its a frustration ive run into myself many times